examples/simple.py

```diff
@@ -24,7 +24,7 @@ def main():
              Item('Name2', 'Description2'),
              Item('Name3', 'Description3')]
 
-    table = ItemTable(items)
+    table = ItemTable(items, border=True)
```

Comment:
I would suggest that you remove this from here and add some information to the README.

This example is intended to be the simplest working example.